### PR TITLE
for 4 and 5 make SQ_GOOD if have on first page

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

**Comment by davgur** [comment]
[Eval expectations new logic](https://trello.com/c/FQHt322Y/119-eval-expectations-new-logic)

File: search/quality_eval.go [hunk 1]
```diff
@@ -698,7 +698,7 @@ func EvaluateQuery(q EvalQuery, serverUrl string, skipExpectations bool) EvalRes
 					}
 					if HitMatchesExpectation(hit, hitSource, q.Expectations[i]) {
 						rank = j + 1
-						if j <= 2 {
+						if j <= 2 || i > 2 {
```
Comment (YuriHechter) [inline search/quality_eval.go:701]: please add a comment that describes the new logic

Comment (YuriHechter) [inline search/quality_eval.go:701]: This condition is not enough. Ranks 6-9 also became 'good'.

Comment (davgur) [reply search/quality_eval.go]: added

Comment (davgur) [reply search/quality_eval.go]: fixrd

**Pull request merged**